pkg/cidata/cidata.TEMPLATE.d/boot.sh

```diff
@@ -77,6 +77,64 @@ if [ -d "${LIMA_CIDATA_MNT}"/provision.data ]; then
 	done
 fi
 
+if [ -d "${LIMA_CIDATA_MNT}"/provision.yq ]; then
+	yq="${LIMA_CIDATA_GUEST_INSTALL_PREFIX}/bin/lima-guestagent yq"
+	if ${yq} --version >/dev/null; then
+		for expression in "${LIMA_CIDATA_MNT}"/provision.yq/*; do
+			filename=$(basename "${expression}")
+			format=$(deref "LIMA_CIDATA_YQ_PROVISION_${filename}_FORMAT")
+			path=$(deref "LIMA_CIDATA_YQ_PROVISION_${filename}_PATH")
+			permissions=$(deref "LIMA_CIDATA_YQ_PROVISION_${filename}_PERMISSIONS")
+			user=$(deref "LIMA_CIDATA_YQ_PROVISION_${filename}_USER")
+			if ! sudo -iu "${user}" mkdir -p "$(dirname "${path}")"; then
+				WARNING "Failed to create directory for ${path} (as user ${user})"
+				CODE=1
+				continue
+			fi
+			# Since CIDATA is mounted with dmode=700,fmode=700,
+			# provision.tool/yq cannot be executed by non-root users,
```

Comment:
This isn't actually true anymore because the guestagent gets installed, but see comment above about calling the guestagent from the cidata mount for plain instances, which makes it true again (except for the command name).

```suggestion
			# `lima-guestagent yq` cannot be executed by non-root users,
```